tensorflow-core/tensorflow-core-api/build.sh

```diff
@@ -16,12 +17,17 @@ else
     export PYTHON_BIN_PATH=$(which python3)
 fi
 
+# Add platform specific flags
+if [[ "${PLATFORM:-}" == macosx-arm64 ]]; then
+  BUILD_FLAGS="$BUILD_FLAGS --config=macos_arm64"
+fi
+
 if [[ "${EXTENSION:-}" == *mkl* ]]; then
-    export BUILD_FLAGS="$BUILD_FLAGS --config=mkl"
+    BUILD_FLAGS="$BUILD_FLAGS --config=mkl"
```

Comment:
Should we strip all the MKL bits at this point?

Reply:
As being discussed [here](https://github.com/tensorflow/java/issues/394#issuecomment-1354030460), I'm planning to do that in a separate PR, since this one already include enough changes.